### DISCUSSION TITLE
Support for v2 API?

### DISCUSSION THREAD

**Discussion by wishborn** [openai-php/client#380] [Q&A] [answered]
I just noticed that OpenAI has a version 2 of their API and my assistants have already been migrated to the new version. Does this package support their v2 changes and if so, how can I enable them?

**Comment by gehrisandro** [comment, marked as answer]
Not yet, going to do it asap.

**Reply by knash94** [reply]
Hey @gehrisandro.

Has there been any progress made on this to date? If not then I'm able to spend some time this week supporting v2 as a fork from #367. Hopefully, I'll be able to update the existing endpoints and add support for the vector stores APIs

**Comment by gehrisandro** [comment]
API v2 support comes in this PR: https://github.com/openai-php/client/pull/405

Feel free to check it out and give feedback there (atm, only the vector stores API is implemented, the rest is coming the next days).

**Comment by gehrisandro** [comment]
Release for assistants v2 has arrived (beta). 🚀

Happy to hear your feedback before making it a stable release: https://github.com/openai-php/client/discussions/411

https://github.com/openai-php/client/releases/tag/v0.10.0-beta.1
